bentoctl/deployment_config.py

```diff
@@ -199,6 +200,9 @@ def create_deployable(self, destination_dir=os.curdir):
         for building
         """
         bento_metadata = get_bento_metadata(self.bento.path)
+        # In the case of debug mode, we want to keep the deployable for debugging purpose.
+        # So by setting overwrite_deployable to false, we don't delete the deployable after the build.
+        overwrite_deployable = not get_debug_mode()
         (
```

Comment:
even with this flag the deployable is currently created in the tempdir right? should we move it over to the current directory (only in debug mode) since there is always the temp directory might get cleared

Reply:
No, we should try created in the temp dir, the bentoctl developers and usder who needs to debug can go find it. Also we can turn off the clean, if the debug is true